src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -5212,8 +5212,8 @@ public async Task FormatRecursivePattern_PreferSpaceInsideEmptyPositionalSubpatt
 {
     void M()
     {
-        _ = this is C(1, 2) { };
-        _ = this is C( ) { };
+        _ = this is C(1, 2) {};
+        _ = this is C( ) {};
```

Comment:
There probably shouldn't be a space inside `( )` right?

Reply:
This is due to option `SpaceBetweenEmptyMethodCallParentheses` above